codegen/projections/white_label/lib/white_label/client.rb

```diff
@@ -9,6 +9,9 @@
 
 require 'stringio'
 
+require_relative 'middleware/test_middleware'
```

Comment:
We should add something to the way we write out the dependent ruby files (eg when doing `.rubySource("smithy-ruby-codegen-test-utils/middleware/test_middleware.rb")` to remove duplicates - IE, probably just using a Set is sufficent.